### ISSUE TITLE
Pretext accuracy of RotNet with AlexNet on ImageNet?

### ISSUE THREAD

**Issue by jasonwei20** [gidariss/FeatureLearningRotNet#17]
What is the rotation prediction accuracy of RotNet using AlexNet trained on ImageNet

**Comment by jasonwei20** [comment]
Also, does better rotation prediction accuracy correlate with better accuracy on the downstream task? In Figure 5(a) in the paper, the downstream accuracy did not improve after 40 epochs, although the rotation accuracy did improve.

**Comment by gidariss** [comment]
Hi Jason,

The rotation prediction accuracy of a AlexNet-based RotNet on the validation set of ImageNet is around 89.0%. 

Regarding the correlation question, in general (and I believe this is a common observation among all self-supervised methods) training for longer on the pretext task (which usually leads to improved  accuracy on the pretext task) leads to improved accuracy on the downstream task.
